### PR TITLE
:robot: Add dnspyex.vm

### PR DESCRIPTION
**Pull request by vm-packages**
Automated PR from https://github.com/mandiant/VM-Packages/issues/66 :rocket:

### CODE REVIEW

File: pkg.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (Ana06) [inline pkg.json:1]: This needs to be removed: https://github.com/mandiant/VM-Packages/pull/69/files#r968504190

**Review by Ana06 — CHANGES_REQUESTED** [review]

**Comment by Ana06** [comment]
why is this `scripts/utils/__pycache__/create_package_template.cpython-38.pyc` generated @mr-tz ? 😕 

We may want to:
1. ignore `.pyc` files
2. Instead of deleting created files in the action, commit only the package directory with `add-paths`

**Comment by mr-tz** [comment]
Ugh, yes, let's explicitly add the created files. I'll take a look into it.

**Review by Ana06 — APPROVED** [review]
This finally looks good!!! 🎉

**Pull request merged**